android-components/components/ui/icons/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<resources>
```

Comment:
Specifically, I am expecting that we set these color variables according to what we want in the app instead of inside of the ui-icons component. This is similar to https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/res/values/colors.xml#348.

Reply:
Ported the colors to Fenix, but there is no photon color for the light unfilled color (D9D9D9)